src_c/polygon.c

```diff
@@ -94,7 +112,8 @@ pgPolygon_FromObject(PyObject *obj, pgPolygonBase *out)
 
         memcpy(out->vertices, poly->vertices,
                poly->verts_num * 2 * sizeof(double));
-
+        /* Function to set polygon center*/
```

Comment:
already explained by the name of the functions, dont add comments if the function name already covers it